package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "_s",
+  "version": "1.0.0",
+  "license": "GPL-2.0+",
+  "scripts": {
+    "build:css": "npm run compile:css && npm run prefix:css && npm run format:css && npm run fix:css",
+    "compile:css": "node-sass sass/style.scss style.css --output-style expanded --indent-type tab --indent-width 1",
+    "prefix:css": "postcss style.css -u autoprefixer -o style.css --no-map",
+    "lint:scss": "stylelint 'sass/**/*.scss' --syntax scss --config node_modules/stylelint-config-wordpress/scss.js",
+    "lint:css": "stylelint style.css",
+    "fix:scss": "stylelint 'sass/**/*.scss' --syntax scss --fix --config node_modules/stylelint-config-wordpress/scss.js",
+    "fix:css": "stylelint style.css --fix",
+    "format:css": "stylefmt style.css"
+  },
+  "devDependencies": {
+    "autoprefixer": "^7.1.6",
+    "browserslist": "^2.9.0",
+    "node-sass": "^4.6.1",
+    "postcss-cli": "^4.1.1",
+    "stylefmt": "^6.0.0",
+    "stylelint": "^8.2.0",
+    "stylelint-config-wordpress": "^12.0.0"
+  },
+  "stylelint": {
+    "defaultSeverity": "warning",
```

Comment:
@grappler Not sure if this is wanted here?
I added it to get rid of errors printed in the terminal when linting didn't return 0 errors.
```
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! _s@1.0.0 fix:css: `stylelint style.css --fix`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the _s@1.0.0 fix:css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/marty/.npm/_logs/2017-11-15T15_48_26_137Z-debug.log
error Command failed with exit code 2.
```

Reply:
I am fine with leaving like this. We can look in the future to change the command to something like `"stylelint '**/*.scss'; exit 0"` if need be.

https://medium.com/@bjankord/how-to-lint-scss-with-stylelint-dc87809a9878